VideoLocker/res/values/strings.xml

```diff
@@ -99,7 +99,7 @@
     <string name="confirm_dialog_message_help">Enter the e-mail address for your account, and we\'ll send you instructions to
          reset your password.</string>
     <string name="wifi_dialog_message_help">Allow your device to download videos over your cellular connection when 
-        Wi-Fi is not available. Data charges may apply.</string>
+        \nWi-Fi is not available. Data charges may apply.</string>
```

Comment:
Shouldn't the UI be responsible for the line breaks? This will do funny things in translation